stackit/internal/services/argus/argus_acc_test.go

```diff
@@ -236,7 +310,16 @@ func TestAccResource(t *testing.T) {
 			},
 			// Update
 			{
-				Config: resourceConfig(fmt.Sprintf("%s-new", instanceResource["name"]), instanceResource["new_plan_name"], "", "true"),
+				Config: resourceConfig(utils.Ptr(fmt.Sprintf(
+					"[%q, %q]",
+					instanceResource["acl-0"],
+					instanceResource["acl-1-updated"],
+				)),
+					fmt.Sprintf("%s-new", instanceResource["name"]),
+					instanceResource["new_plan_name"],
+					"",
+					"true",
+				),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					// Instance
 					resource.TestCheckResourceAttr("stackit_argus_instance.instance", "project_id", instanceResource["project_id"]),
```

Comment:
We should also add checks here to test that the ACLs are set and updates correctly